### PR TITLE
Update Korean Translation (2023-03-01)

### PR DESCRIPTION
**Pull request by jooyunghan**

### CODE REVIEW

File: po/ko.po [hunk 1]
```diff
@@ -14076,6 +14284,27 @@ msgid ""
 "```"
 msgstr ""
 
+#~ msgid ""
```
Comment (jiyongp) [inline po/ko.po:14287]: Why we need these?

File: po/ko.po [hunk 2]
```diff
@@ -6937,19 +7087,29 @@ msgid ""
...
+"}\n"
+"```"
 
 #: src/std/hashmap.md:38
 msgid ""
 "* `HashMap` is not defined in the prelude and needs to be brought into scope.\n"
 "* Try the following lines of code. The first line will see if a book is in the hashmap and if not return an alternative value. The second line will insert the alternative value in the hashmap if the book is not found."
 msgstr ""
+"* `HashMap`은 prelude에 정의되어 있지 않기 때문에 명시적으로 추가해줘야 합니다.\n"
+"* 아래 코드를 테스트해보세요. 첫 문장에서는 해시맵에 책이 있는지 검사하여 없으면 디폴트 값을 반환합니다. 두번 째 문장에서는 해시맵에 해당 책이 없는 경우 값을 해시맵에 값을 추가합니다."
```
Comment (jiyongp) [inline po/ko.po:7112]: Is "문장" the well-recognized translation for "statement"? The Rust book used "구문" and I'd like to follow them.

```suggestion
"* 아래 코드를 테스트해보세요. 첫 구문에서는 해시맵에 책이 있는지 검사하여, 없으면 디폴트 값을 반환합니다. 그 다음 구문에서는 해시맵에 해당 책이 없는 경우, 대체할 값을 해시맵에 일단 넣고, 그 값을 반환합니다."
```

Comment (jooyunghan) [reply po/ko.po:7112]: "구문" is used typically for "syntax" like "구문 분석기", while "-문" is used for various types of "statements": 대입문, for문, 조건문, etc. I think "문장" here is okay, or "대입문" seems okay too.

Comment (jiyongp) [reply po/ko.po:7112]: Ok fine. I don't have a strong opinion in either way. Feel free to revert mine.

File: po/ko.po (hunk 1 above)
Comment (jooyunghan) [reply po/ko.po]: Removed.

**Review by jiyongp — APPROVED** [review]

**Pull request merged**